File: PRACTICE/MODULES/C++/daily_practice/13-09-2024/Association/Student.cpp
#include<iostream>
#include "Address.cpp"
using namespace std;

class Student {
	private:
		int rollNo;
		string name;
		Address address;
	public:
		Student() : rollNo(0), name("") {}
		Student(int rollNo, string name, Address address) : rollNo(rollNo), name(name), address(address) {}
		void Accept() {
			cout<<"enter student details : "<<endl;
			cout<<"roll no is : "<<endl;
			cin>>rollNo;
			cout<<"name is : "<<endl;
			cin>>name;
			address.Accept();
		}
		void Display() {
			cout<<"Student data is : "<<endl;
			cout<<"roll no is : "<<rollNo<<" name is : "<<name<<endl;
			address.Display();
		}
};

int main() {
	Student s;
	s.Accept();
	s.Display();
	return 0;
}

File: PRACTICE/MODULES/OOPsWithJava/DailyPractice/24-09-2024/pnr-240840120156/src/studentdemo/Student.java
package studentdemo;

public class Student {
	int stuId = 1000;
	String stuName;
	double marks;
	
	public Student() {
	
	}
	public Student(String stuName, double marks) {
		this.stuId = stuId++;
		this.stuName = stuName;
		this.marks = marks;
	}
	public void display() {
		System.out.println("Id is : " + stuId);
		System.out.println("Name is : "+ stuName);
		System.out.println("marks is : "+ marks);
	}
}
